### DISCUSSION TITLE
Welcome to ToyBox Discussions!

### DISCUSSION THREAD

**Discussion by cabarius** [cabarius/ToyBox#19] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tyreal2004** [comment]
Any way to edit weapons and armor to add enchantments etc? If not how can I access and edit blueprints?

**Reply by GodIsSoGreat** [reply]
Would also like to know how to change the internal values of items. Like armor% or dodge%. I found the Loot & Pick section and the exact values, but they don't seem editable. Trying to change the damageabsorption field.

**Comment by Zanzy21** [comment]
Was wondering if anyone knew if you can turn off the mythic transformations by etude?

Specifically looking to suppress the demon tail until later in the game as it's ugly and literally unplayable as is.

There used to be a Visual Adjustments mod that could do this but it hasn't been updated in a few patches.

**Comment by rnel76** [comment]
hi, i was wondering if there will be an update for 1.4.0y and if there is an ETA?

**Reply by Zanzy21** [reply]
There is an update. If you look on nexus mods in the posts section there is a link to the working 1.4.0y version on google drive. 

https://drive.google.com/file/d/1IVVoBXBNHmcytmll2IxR0ehvUCSHMrv0/view?usp=sharing

**Reply by rnel76** [reply]
ahh thank you! appreciate the work you do!

**Reply by Zanzy21** [reply]
Not my work. Just someone passing by!

**Comment by LantaVey** [comment]
Hi. I would like to get permission to publish a translation into Russian. On the Nexus website. What permissions can I use?

**Comment by cabarius** [comment]
you are welcome to do do this, consider this your permission.  Please do not do it for profit.  Actually why don't you submit a pull request for the RU localization and then people can download the main version and get all the latest updates automatically?

**Reply by LantaVey** [reply]
Thank you. I'm not going to monetize it. I just want people in my country to be able to use your program while playing this wonderful game.

**Comment by Sturmund-Drang** [comment]
Can somebody please tell me step by step how to search for an item and add it to a character's inventory in the newest version of the Toybox. I don't seem to be able to do it easily and without explanation and now it seems like this feature doesn't work. I am referring to I think Search n' Pick

**Comment by narrowminded56** [comment]
Hello, I would really appreciate if you could implement a feature that allows the player to right click on a character portrait to display options to open inventory, spellbook and character sheet just like it was possible in Pathfinder : Kingmaker. I don't know why the devs didn't implement this function to WOTR. Thank you for your work on this game.

**Comment by Dolphinwithinternet** [comment]
Is there a kingmaker version of this?
